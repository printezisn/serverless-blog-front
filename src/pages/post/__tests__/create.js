import React from "react"
import renderer from "react-test-renderer"

jest.mock("../../../components/layout", () => "Layout")
jest.mock("../../../components/post", () => {
    return {
        EditPost: "EditPost",
    }
})

import Create from "../create"

describe("Create page", () => {
    it("renders correctly", () => {
        const page = renderer.create(<Create />)

        expect(page.toJSON()).toMatchSnapshot()
    })
})
