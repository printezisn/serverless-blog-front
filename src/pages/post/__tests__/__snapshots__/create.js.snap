// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create page renders correctly 1`] = `
<Layout>
  <article>
    <header>
      <h2
        className="title is-4 has-text-centered"
      >
        New Post
      </h2>
    </header>
    <EditPost />
  </article>
</Layout>
`;
